.github/workflows/tests_linux.yml

```diff
@@ -59,27 +59,46 @@ jobs:
               -DPL_BACKEND=${{ matrix.pl_backend }} \
               -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION) \
               -DENABLE_COVERAGE=ON
+
+            cmake . -BBuildKernelOMP -G Ninja \
+              -DCMAKE_BUILD_TYPE=Debug \
+              -DBUILD_TESTS=ON \
+              -DENABLE_PYTHON=OFF \
+              -DPL_BACKEND=${{ matrix.pl_backend }} \
+              -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION) \
+              -DENABLE_COVERAGE=ON \
+              -DLQ_ENABLE_KERNEL_OMP=ON
```

Comment:
@mlxd what are your thoughts on building the lightning sim with OMP kernel support on Catalyst? Do you have any argument against it?

Reply:
I think it depends very much on needs. If you are never using the OpenMP adjoint pipeline through Catalyst, this may be a better option. If you are using the adjoint pipeline with multiple observables, I'd leave this disabled. OpenMP does allow us to scale, but we get much better throughput with the single-thread per circuit pass, isnce the kernels are really optimised for 1-to-many workloads, with a single thread per task.

Reply:
we use the OMP adjoint pipeline. I see the issue with throughput. so, let's leave this disabled then for now, but will add a note to explore this a little bit more in Catalyst after finishing the benchmark suite.